R/g.getmeta.R

```diff
@@ -607,6 +607,41 @@ g.getmeta = function(datafile, params_metrics = c(), params_rawdata = c(),
           }
           CWav[h,1] = max(c(CW[h,1],CW[h,2],CW[h,3])) #indicator of clipping
         }
+        # Revise edges of nonwear score
```

Comment:
When I run this branch on the example data you share with me I get an error:
`Error in data[from:to, ] : subscript out of bounds`

Reply:
I thought I had fixed this, but I am looking at it now, thanks!

Reply:
This should be fixed now